src/stories/containers/Endgame/EndgameContainer.tsx

```diff
@@ -63,8 +63,8 @@ const EndgameContainer: React.FC<EndgameContainerProps> = ({ budgetTransitionAna
       </Container>
 
       <div ref={keyChangesRef}>
-        <BannerContainer id="section-key-changes">
-          <EndgameIntroductionBanner isKeyChanges />
+        <BannerContainer>
```

Comment:
Please, note that if you remove the `id` the navigation is not going to work properly

Reply:
you are right, done